server/src/main/java/org/spine3/server/event/enrich/EventEnricher.java

```diff
@@ -227,8 +227,9 @@ private Builder() {}
             return this;
         }
 
-        public <M extends Message, E extends Message> Builder addFieldEnrichment(Class<M> eventMessageClass,
-        Class<E> enrichmentClass, Function<M, E> function) {
+        public <M, E> Builder addFieldEnrichment(Class<M> eventMessageClass,
```

Comment:
If it comes to field enrichment, the parameter should be probably called `fieldClass` instead of `eventMessageClass`.

Consider using `S` and `T` generic type names as you suggested.